src/ucp/rndv/rndv_ats.c

```diff
@@ -24,7 +24,7 @@ ucp_proto_rndv_ats_init(const ucp_proto_init_params_t *init_params)
 
     *init_params->priv_size = sizeof(ucp_proto_rndv_ack_priv_t);
 
-    caps.cfg_thresh   = 0;
+    caps.cfg_thresh   = UCS_MEMUNITS_AUTO;
```

Comment:
why?

Reply:
With this patch, RNDV variants are compared together with eager one, so setting 0 here would force empty ATS over short which is not good decision for performance.